setup.py

```diff
@@ -49,6 +49,13 @@ def read(*rnames):
       package_dir={'': 'src'},
       install_requires=[
           'setuptools',
+
+      ],
+      setup_requires=[
+          'pytest-runner',
```

Comment:
Ah thank you for the link. Looks like it is needed to make something like this working: `python setup.py test`.
I was asking, because for me it looks like that when RestrictedPython is installed also pytest-runner gets installed (correct me if I'm wrong). So when RestrictedPython is a dependency of another package (for example Zope2), then pytest-runner gets also installed, even if its only used for tests !
From my point of view stuff which gets installed outside of testing should be as minimal as possible.